Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '2.1.2'
+ruby '2.1.4'
```

Comment:
Please don't change the ruby version in the pull request (you can use `git add -p` to explicitly add individual patches to the staged index before commiting).